.circleci/config.yml

```diff
@@ -296,7 +296,7 @@ commands:
     steps:
       - run:
           name: Compiling TypeScript
-          command: yarn workspaces foreach --topological-dev --verbose run compile
+          command: yarn workspaces foreach --exclude fxa-payments-server --exclude fxa-content-server --topological-dev --verbose run compile
```

Comment:
Do we still need this since the changes below are fixing the errors?

Reply:
It doesn't fix all unfortunately, payments server had alot of things but I can file an issue for them to prioritize.